internal/app/agent/service.go

```diff
@@ -223,14 +257,82 @@ func (a *agent) ServiceConfig(uuid, cmdStr string) error {
 	return a.processResponse(uuid, cmd, resp)
 }
 
-func (a *agent) processResponse(uuid, cmd, resp string) error {
-	payload, err := encodeSenML(uuid, cmd, resp)
+func (a *agent) Terminal(uuid, cmdStr string) error {
+	b, err := base64.StdEncoding.DecodeString(cmdStr)
 	if err != nil {
-		return err
+		return errors.New(err.Error())
+	}
+	cmdArgs := strings.Split(string(b), ",")
+	if len(cmdArgs) < 1 {
```

Comment:
`if len(cmdArgs) < 2 `

Reply:
this is actually ok, `open` and `close` dont have additional elemtns